runner.py

```diff
@@ -329,10 +330,12 @@ def runner(input_dir, reference_file, output_dir, max_basecall_iterations, min_c
                                       mode=mode, num_alignments=num_alignments, opposing_strings=overlapping_reads,
                                       output_dir=output_dir, perc_identity=perc_identity, reference_file=reference_file,
                                       processing_dir=filenames['processing_dir'], quality_threshold=quality_threshold,
-                                      task=task, basecall_dir=filenames['basecall_dir'])
+                                      task=task, basecall_dir=filenames['basecall_dir'],
+                                      is_overlapping=overlapping_reads)
```

Comment:
why duplicate an existing variable under a different name can cause confusion.
for example, I actually already put in that variable under the name opposing_strings right here in line 330 and because it is under a different name you actually put it again under yet another name..!!!
This is a good opportunity to fix that as well... thanks!